X509.pm

```diff
@@ -172,6 +173,11 @@ sub Crypt::OpenSSL::X509::subjectaltname {
   die '*** Could not prepare definition: '.$asn->error()
       if !$ok;
 
+  # Microsoft's User Principal Name (Smart Card Logon)
+  my $upn = Convert::ASN1->new or die( "New UPN" );
+  $upn->prepare(q(microsoftUPN UTF8String)) or die ( "Prepare UPN" );
```

Comment:
Let's make it `q(UPN)` (drop the 'microsoft') ; agree that consistency with openssl's decode is a good thing.

I should have checked that; good catch.